web/src/components/modals/time-range-modal.tsx

```diff
@@ -271,6 +279,16 @@ export const TimeRangeModal: React.FC<TimeRangeModalProps> = ({ id, isModalOpen,
         </Text>
       </TextContent>
       {getDateTimePickers()}
+      {!Number.isNaN(maxChunkAge) && (
+        <TextContent>
+          <Text component={TextVariants.blockquote}>
+            {t(
+              'Collection latency could be up to {{maxChunkAge}} corresponding to the current Loki "max_chunk_age" ingester configuration.',
```

Comment:
what exactly is the intent of this message on this dialog? Users can select a time range bigger than the max chunk age, right?

Reply:
The goal is to inform users that flows/metrics can still appear during the max chunk age = 2h for loki operator.
According to our last discussion, we decided to put the message in the time frame selection.